src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -35,6 +35,22 @@ constexpr const auto UpdatePatternLocationsInterval = std::chrono::milliseconds(
 
 namespace winrt::Microsoft::Terminal::Control::implementation
 {
+    // Helper to check if we're on Windows 11 or not. This is used to check if
+    // we need to use acrylic to achieve transparency, because vintage opacity
+    // doesn't work in islands on win10.
+    // Remove when we can remove the rest of GH#11285
+    static bool _isVintageOpacityAvailable() noexcept

```

Comment:
Do we want to use a magic static here to save time? Or is this a fairly cheap thing to do?

Reply:
I flipped a coin to see if anyone would nit on this. Turns out I was wrong 🤣

Reply:
It's super cheap.